modules/hibernating-cluster-hibernate.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/hibernating-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hibernating-cluster-hibernate_{context}"]
+= Hibernating a cluster
+
+You can hibernate a cluster for up to 90 days. The cluster can recover if certificates expire while the cluster was in hibernation.
+
+.Prerequisites
+
+* The cluster has been running for at least 24 hours to allow the first certificate rotation to complete.
++
+If you must hibernate your cluster before the 24 hour certificate rotation, then use the following procedure instead: link:https://www.redhat.com/en/blog/enabling-openshift-4-clusters-to-stop-and-resume-cluster-vms[Enabling OpenShift 4 Clusters to Stop and Resume Cluster VMs].
+
+* You have access to the cluster as a user with the `cluster-admin` role.
++
+// TODO: Confirm must be a cluster-admin
```

Comment:
@vrutkovs Can you confirm that they'd need to be cluster admin to hibernate/resume (to be able to see cluster operators/other resources)?

Reply:
It needs to be an admin user able to approve CSRs, check node and operator states. Its a mouthful to mention all permissions required so lets use cluster-admin here